hedera-node/hedera-app/src/main/java/com/hedera/node/app/fees/congestion/MonoMultiplierSources.java

```diff
@@ -45,7 +48,17 @@ public void updateMultiplier(@NonNull final Instant consensusTime) {
             // only accessor.congestionExempt() is used in the mono multiplier implementation and that seems like could
             // be only set for triggered transactions
             // so using just a dummy accessor
-            accessor = SignedTxnAccessor.from(Bytes.EMPTY.toByteArray());
+            final var dummySignedTxn = SignedTransaction.newBuilder()
+                    .bodyBytes(TransactionBody.PROTOBUF.toBytes(TransactionBody.newBuilder()
+                            .cryptoTransfer(
+                                    CryptoTransferTransactionBody.newBuilder().build())
+                            .build()))
+                    .build();
+            final var dummyTxn = Transaction.newBuilder()
```

Comment:
I believe this can be a static property, and we do not need to generate it for each call.

Reply:
Ah, maybe better not. `SingedTxnAccessor` is mutable. I do not think it matters here, but it always feels weird to create a static member that is mutable... 🤔

Reply:
Updated.